defaults/preferences/thunderlink.js

```diff
@@ -19,6 +20,7 @@ pref("extensions.thunderlink.custom-tl-string-1-appendtofile-path", "");
 
 pref("extensions.thunderlink.custom-tl-string-2-title", "local cmd");
 pref("extensions.thunderlink.custom-tl-string-2", "<tbexe> <thunderlink>");
+pref("extensions.thunderlink.custom-tl-string-2-selection-delimiter", "");
```

Comment:
for consistency these should probably all be the same `" / "` - 7 and 8 below are also the empty string while the rest are `" / "`

Reply:
ok

Reply:
You got 7 and 8 but 2 is still the empty string :-)

Reply:
I just pulled this to test it, and the 2nd prefs string is the last bit - it's still empty, after that this looks ready to go

Reply:
fixed in new commit